zeekpkg/manager.py

```diff
@@ -3201,12 +3204,14 @@ def _is_branch_outdated(clone, branch):
 def _is_clone_outdated(clone, ref_name, tracking_method):
     if tracking_method == TRACKING_METHOD_VERSION:
         return _is_version_outdated(clone, ref_name)
-    elif tracking_method == TRACKING_METHOD_BRANCH:
+
+    if tracking_method == TRACKING_METHOD_BRANCH:
         return _is_branch_outdated(clone, ref_name)
-    elif tracking_method == TRACKING_METHOD_COMMIT:
+
+    if tracking_method == TRACKING_METHOD_COMMIT:
         return False
```

Comment:
Might be very subjective, but the `if..elif..elif` seemed nicer . I guess this should be `match case` anyhow if we can, so just ignore this drive-by comment :-)

Reply:
Ah, Python 3.9. Sorry for the noise.. I'm resolving, do what you think makes sense :-)